Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -80,6 +80,7 @@ class BrowserViewController: UIViewController {
     private var scrollController = BrowserScrollingController()
 
     private var keyboardState: KeyboardState?
+    private var isInBackground = false
```

Comment:
Assuming we still want this check, could we replace this flag with `UIApplication.sharedApplication().applicationState == .Background`?